projects/demo/src/app/cds-theme-select.component.ts

```diff
@@ -35,6 +39,12 @@ export class CdsThemeSelectComponent implements OnInit, OnDestroy {
     this.applyTheme(null);
   }
 
+  onThemeChange(event: SelectChangeEvent) {
+    if (event?.target) {
+      this.applyTheme(event.target.value); // may have to cast `target` to `any` here
```

Comment:
The comment should be removed.

Reply:
Also, this commit is a chore, not a fix, because it doesn't affect any library.

Reply:
Also, I'm still curious why this is needed. How can this issue be reproduced?

Reply:
Runtime error. Pull up the `main` branch and run it :)

Reply:
I unresolved this because I can produce an error. Could you provide any more detailed reproduction steps? I am mainly curious as to how this error is possible.

Reply:
Strange. Getting it again and running it doesn't raise an error. Hard to reproduce. I'll remove the commit. At first I had the same impression as you - Why would that be an issue? But the runtime was raising an issue in the devtools and it was stopping me to continue on the rest.